webrtc.html

```diff
@@ -1586,22 +1586,20 @@
                         "applying-a-remote-desc">[[!JSEP]]</span>.</p>
                       </li>
                       <li>
-                        <p>If the direction of the media description as defined
-                        in
-                        <span data-jsep="applying-a-remote-desc">[[!JSEP]]</span>
-                        is <code>sendrecv</code> or <code>sendonly</code>, and
+                        <p>If <var>direction</var> is <code>sendrecv</code> or
+                        <code>recvonly</code>, and
```

Comment:
@taylor-b I reversed it to match github tip's [**set an RTCSessionDescription**](https://rawgit.com/w3c/webrtc-pc/master/webrtc.html#set-description)'s step 2.2.8.1., which says:
 * *"Let* direction *be an RTCRtpTransceiverDirection value representing the direction from the media description, but with the send and receive directions reversed to represent this peer's point of view."*

So unless I'm royally confused, wouldn't a remote offer with incoming media have "sendonly" and cause *direction* to be *recvonly* here?